administrator/components/com_banners/src/Model/ClientModel.php

```diff
@@ -134,5 +134,19 @@ protected function loadFormData()
     protected function prepareTable($table)
     {
         $table->name = htmlspecialchars_decode($table->name, ENT_QUOTES);
+
+
+        $app = \Joomla\CMS\Factory::getApplication();
+
+        $existing = $this->getTable();
+        $existing->load(['name' => $table->name]);
+
+        //show a warning
```

Comment:
```suggestion
        // Show a warning
```
Code style: `//`should always be followed by a space if not used to temporarily disable some code.